airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/http_requester.py

```diff
@@ -33,74 +34,55 @@
 
 
 @dataclass
-class HttpRequester(Requester):
+class HttpRequester(Requester, ABC):
     """
-    Default implementation of a Requester
+    Abstract default implementation of a Requester
 
     Attributes:
         name (str): Name of the stream. Only used for request/response caching
-        url_base (Union[InterpolatedString, str]): Base url to send requests to
-        path (Union[InterpolatedString, str]): Path to send requests to
         http_method (Union[str, HttpMethod]): HTTP method to use when sending requests
-        request_options_provider (Optional[InterpolatedRequestOptionsProvider]): request option provider defining the options to set on outgoing requests
         authenticator (DeclarativeAuthenticator): Authenticator defining how to authenticate to the source
         error_handler (Optional[ErrorHandler]): Error handler defining how to detect and handle errors
-        config (Config): The user-provided configuration as specified by the source's spec
     """
 
     name: str
-    url_base: Union[InterpolatedString, str]
-    path: Union[InterpolatedString, str]
-    config: Config
     parameters: InitVar[Mapping[str, Any]]
```

Comment:
can we move this parameter to the low code implementation? There shouldn't be any reason to keep this empty parameters field in the parent class

Reply:
Done